packages/survey-creator-core/src/localization/english.ts

```diff
@@ -715,14 +827,13 @@ export var enStrings = {
     asc: "Ascending",
     desc: "Descending",
     indeterminate: "Indeterminate",
+    selected: "Selected",
+    unselected: "Unselected",
     decimal: "decimal",
     currency: "currency",
     percent: "percent",
     firstExpanded: "First panel is expanded",
     off: "Hide question numbers",
-    onpanel: "Start on each panel",
-    onPanel: "Start on each panel",
-    onSurvey: "Continue throughout the survey",
```

Comment:
Hi @RomanTsukanov, I think this change should have been considered breaking because it has required us to move our (re-)localised stuff down a level to `showQuestionNumbers`.

That's fine for us (fortunately our type checking picker it up), but could cause issues for other users whose `onSurvey` localisation is no longer being applied how they think it is.

Thanks :)

Reply:
Hi @RobHannay,

You are correct, it is indeed a breaking change. We'll update our [release notes](https://surveyjs.io/stay-updated/release-notes) accordingly to notify other users about it. Thanks for bringing this to our attention.